### PR TITLE
Upcoming & ongoing on the dashboard

### LINKED ISSUE #495: "Upcoming" workshops list
I like having this list displayed on the AMY dashboard.  Can you change it so workshops remain on the list until the workshop is over -  so it should display all workshops where end_date >= today.

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This fixes #495.

The fix was pretty easy thanks to the QuerySet logic.

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
@gvwilson awaiting feedback on this PR and #496 (but I need to rebase it first) and #494 and #493.

File: workshops/templates/workshops/dashboard.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
   <div class="col-sm-8">
     <div class="row">
       <div class="col-xs-3">
-        <h3>Upcoming</h3>
+        <h3>Upcoming & ongoing</h3>
```
Comment (gvwilson) [inline workshops/templates/workshops/dashboard.html]: `&amp;` not raw `&`

File: workshops/test/base.py [hunk 2]
```diff
@@ -221,7 +221,7 @@ def _setUpEvents(self):
         event_end = today + datetime.timedelta(days=1)
         Event.objects.create(start=event_start,
                              end=event_end,
-                             slug='ends_tomorrow',
+                             slug='ends_tomorrow-ongoing',
```
Comment (gvwilson) [inline workshops/test/base.py]: is this `ends_tomorrow - ongoing` (i.e., is that a dash or a minus)?

File: workshops/test/base.py [hunk 3]
```diff
@@ -231,7 +231,7 @@ def _setUpEvents(self):
         event_end = today
         Event.objects.create(start=event_start,
                              end=event_end,
-                             slug='ends_today',
+                             slug='ends_today-ongoing',
```
Comment (gvwilson) [inline workshops/test/base.py]: as above - if it's just a join-up dash, please use an underscore (it's confusing to read)

File: workshops/test/base.py [hunk 4]
```diff
@@ -241,7 +241,7 @@ def _setUpEvents(self):
         event_end = today + datetime.timedelta(days=1)
         Event.objects.create(start=event_start,
                              end=event_end,
-                             slug='starts_today',
+                             slug='starts_today-ongoing',
```
Comment (gvwilson) [inline workshops/test/base.py]: as above

File: workshops/test/test_event.py [hunk 5]
```diff
@@ -62,9 +62,9 @@ def test_get_ongoing_events(self):
 
         ongoing_events = Event.objects.ongoing_events()
         event_slugs = [e.slug for e in ongoing_events]
-        correct_slugs = ['starts_today',
-                         'ends_tomorrow',
-                         'ends_today', ]
+        correct_slugs = ['starts_today-ongoing',
+                         'ends_tomorrow-ongoing',
+                         'ends_today-ongoing', ]
```
Comment (gvwilson) [inline workshops/test/test_event.py]: as above - the mix of underscores and dashes is confusing

**Comment by pbanaszkiewicz** [comment]
@gvwilson fixed.

**Comment by gvwilson** [comment]
+1 to merge.

**Pull request merged**